couchbase.py

```diff
@@ -303,6 +304,20 @@ def _parse_metrics(obj_to_parse, dimensions, request_type, module_config):
                     if metric:
                         metrics.append(metric)
 
+        if 'hot_keys' in obj_to_parse:
+            hot_keys = obj_to_parse['hot_keys']
+            key = 0
+            metric_name_pref = 'bucket.hot_keys'
+            for keys in hot_keys:
+                for name, value in keys.items():
+                    if isinstance(value, numbers.Number):
```

Comment:
It seems like it would be better to explicitly reference the `ops` field instead of testing for numbers.  Could the field name potentially change or did you do this for version compatibility?

I'm looking at https://developer.couchbase.com/documentation/server/current/rest-api/rest-bucket-stats.html#story-h2-4.